Meta/check-newlines-at-eof.py

```diff
@@ -0,0 +1,61 @@
+#!/bin/python3
+
+import os
+import subprocess
+import sys
+
+os.chdir(os.path.dirname(__file__) + "/..")
+
+files = subprocess.run(
+    [
+        "git", "ls-files", "--",
+        "*.cpp",
+        "*.h",
+        "*.gml",
+        "*.html",
+        "*.js",
+        "*.css",
+        "*.sh",
+        "*.py",
+        "CMakeLists.txt",
+        "*/CMakeLists.txt",
+        ":!:Base",
+        ":!:Kernel/FileSystem/ext2_fs.h",
+        ":!:Libraries/LibC/getopt.cpp",
+        ":!:Libraries/LibCore/puff.h",
+        ":!:Libraries/LibCore/puff.cpp",
+        ":!:Libraries/LibELF/exec_elf.h"
+    ],
+    capture_output=True
+).stdout.decode().rstrip('\n').split('\n')
```

Comment:
Since we are using `split('\n')`, there is no reason to not use `strip('\n')`, I'll change it.